### PR TITLE
FOGL-5336 - authtoken management route added

### PR DESCRIPTION
**Pull request by ashish-jabble**
Signed-off-by: ashish-jabble <ashish@dianomic.com>

~**NOTE** - I have exposed route in Admin API for intial testing, And definitely its move to management routes~

```
$ curl -sX GET http://localhost:<core_mgt_port>/fledge/service/authtoken | jq
{
  "message": "Resource you were trying to reach is absolutely forbidden for some reason"
}

$ curl -sX GET http://localhost:<core_mgt_port>/fledge/service/authtoken | jq
{
  "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1aWQiOjEsImV4cCI6MTYxOTcxMTg4Nn0.j1ENgEO0jb5NMTty0WzRNDTgQAqJyqTSzGABSgjtWPA"
}
```

### CODE REVIEW

**Review by MarkRiddoch — DISMISSED** [review]
This looks good for getting the auth token, I believe the original ticket stated we need to be sure the service was authenticated and holding a valid bearer token before we give back an oath token. Are you planning to add that later?

**Review by praveen-garg — DISMISSED** [review]
lgtm! 

1. **until** bearer token based on registration token is done/ solidify, we can check and reject authtoken endpoint with 405 if  RemoteAddr port does not belong to management service.  
2. test normal pwd based auth for invalid pwd (may be unit test can handle the case well)

File: python/fledge/services/core/server.py [hunk 1]
```diff
@@ -1230,6 +1231,55 @@ async def get_service(cls, request):
...
+                # Assuming that super admin user exists on the system
+                payload = payload_builder.PayloadBuilder().SELECT("uname", "pwd").WHERE(['id', '=', 1]).payload()
+                result = await cls._storage_client_async.query_tbl_with_payload('users', payload)
+                uid, token, is_admin = await User.Objects.login("admin", result['rows'][0]['pwd'], host)
+            else:
+                # For auth method "any" we can use either login with cert or password
+                token = await cert_login(ca_cert_name)
+                # TODO: if cert does not exist then may try with password
+        except api_exception.AuthenticationIsOptional:
+            raise web.HTTPForbidden
```
Comment (ashish-jabble) [inline python/fledge/services/core/server.py:1277]: Should return proper JSON body as well

**Review by praveen-garg — APPROVED** [review]

**Review by MarkRiddoch — APPROVED** [review]

**Comment by ashish-jabble** [comment]
As discussed with @MarkRiddoch We are merging this PR as bearer token authentication can be added later As this is more for control things we have planned.

**Pull request merged**